File: MPI/mpicheck/tests/functions/4.0/MPI_Type_create_hvector_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Count var_0;
    MPI_Count var_1;
    MPI_Count var_2;
    MPI_Datatype var_3;
    MPI_Datatype *var_4;
    int ret;
    /* calls */
    ret = MPI_Type_create_hvector_c(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Type_create_hvector_c(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Type_create_hvector_c.f

        
        program main
        include 'mpif.h'
        
        INTEGER(KIND=MPI_COUNT_KIND) var_0
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER(KIND=MPI_COUNT_KIND) var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_type_create_hvector_c(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_type_create_hvector_c(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    